search-enterprise/latest/using/index.md

```diff
@@ -232,23 +232,24 @@ In the REST API you can specify the timezone to be used in search for date range
 
 ### Search using date math
 
-Date range queries can be even more powerful when applying date math functions. AFTS supports adding and subtracting periods, as well as rounding:
+Date range queries can be more powerful when applying date math functions. AFTS supports adding and subtracting periods, as well as rounding:
 
-| AFTS query | Description | Notes |
-| ---------- | ----------- | ----- |
-| `acme:projectStartDate:[NOW TO NOW+1DAY>` | Documents with a project start date in the next twenty four hours | |
-| `acme:projectStartDate:[NOW/DAY TO NOW/DAY+1DAY>` | Documents with a project start date today (between midnight and midnight UTC) | Note the subtle difference with the previous query |
-| `acme:projectStartDate:[NOW-1MONTH/YEAR TO NOW-1MONTH/YEAR+1DAY>` | Documents with a project start date in the first day of the year that last month fell in | It's possible to chain date math functions together |
-| `cm:created:[2020-11-01T12:34:00/YEAR TO NOW>` | Documents created since the start of 2020. | It's also possible to apply date math to absolute points in time. |
+| AFTS query | Description |
+| ---------- | ----------- |
+| `acme:projectStartDate:[NOW TO NOW+1DAY>` | Documents that have a project start date in the next twenty four hours. |
+| `acme:projectStartDate:[NOW/DAY TO NOW/DAY+1DAY>` | Documents that have a project start date today and between midnight and midnight (UTC), **Note:** The minor difference between this query and the one above. |
```

Comment:
> Documents that have a project start date today and between midnight and midnight (UTC)
The two clauses are identical - "between midnight and midnight UTC" is just a clarification of "today"

Reply:
| `acme:projectStartDate:[NOW/DAY TO NOW/DAY+1DAY>` | Documents that have a project start date from the current day. The current day is defined as from midnight to midnight (UTC), **Note:** The minor difference between this query and the one above. |

Reply:
I don't think the difference between the two rows is "minor" - I think it's "subtle".  You could just remove the word "minor" completely if you don't want to use "subtle" though.

Reply:
"Documents that have a project start date from the current day" -> "Documents that have a project start date in the current day"